### ISSUE TITLE
32 bit ASK demodulation support

### ISSUE THREAD

**Issue by ozkangoksu** [leech001/RF-BARRIER#2]
Hello. 
I'm trying to implement an RF project to use 32 bit 433mhz RF.

I'm using this RF module:
[433 MHz Module---.pdf](https://github.com/leech001/RF-BARRIER/files/10094956/433.MHz.Module---.pdf)

So the question is; How can I add 32bit remote support to your project?

**Comment by leech001** [comment]
To be honest, I don't really understand what you want. Can you explain in more detail.

**Comment by ozkangoksu** [comment]
I'm trying to develop an RF receiver with my custom made stm32 cpu.

This is the datasheet of the remote: 
[433 MHz Remote Control.pdf](https://github.com/leech001/RF-BARRIER/files/10096440/433.MHz.Remote.Control.pdf)

Remote's RF structure is below:
![image](https://user-images.githubusercontent.com/72453549/204093697-fb480f4c-9d19-4a1a-b15f-e6d469dfbbb0.png)

I tried rc-switch project with an arduino and rc-switch can decode my remote with their protocol 1 as below:

```
04:10:57.252 -> Decimal: 3899580674 (32Bit) Binary: 11101000011011101110000100000010 Tri-State: not applicable PulseLength: 252 microseconds Protocol: 1
04:10:57.389 -> Raw data: 6116,732,240,732,248,728,244,236,736,732,248,236,732,244,732,244,740,240,728,732,244,736,240,244,732,728,244,728,248,52,52,104,40,88,1052,1684,16,1160,32,1680,24,60,268,24,36,36,2992,44,3496,564,68,48,88,452,208,2680,40,40,1328,96,200,68,620,160,52,
```


When I tried your repository, I could not decode my remote because of the timings and I'm asking that how can I add my timings to your project? 

I saw these Receive timings at rf-barrier-timings.h and I'm asking is it possible to use this project with only adding new timings for my remote? I'm just trying to decode the 32 bit RF signal and convert it to decimal format to call some functions. 

```
RF_t hcs301 = {
		.Invert=		1,
		.Te = 			400,
		.TeDrv = 		50,
		.MaxPause = 		20000,
		.CodeLengh = 		66,
		.PreambleLenght = 	23,
		.HeaderLenght=		10
};
```

I'm newbie with stm32 and RF communication. I'm sorry if my questions are silly.
